### DISCUSSION TITLE
How to change hover color of a div element to $primary color dynamically?

### DISCUSSION THREAD

**Discussion by meo1** [quasarframework/quasar#14036] [General - Components / Directives / etc] [answered]
How to change hover color of a div element to $primary color dynamically? 

I have a div element like this:

`<div @click="AddSomething=true" class="aLink">Add Something</div>`

and  in the style section two elements like this:

`.aLink {
  cursor:pointer;
}
.aLink:hover {
 color: $primary;
 font-weight: bold;
}
`
During run of program the user can change theme colors dynamically. It works to some extend, but Quasar does not update theme colors in the "style" section.

What can I do to change hover color of my "div" element dynamically?

**Comment by yusufkandemir** [comment, marked as answer]
Instead of `$primary` which is an SCSS variable, which is static on run-time, you need to use `var(--q-primary)`, which is dynamic on run-time.
https://quasar.dev/style/color-palette#dynamic-change-of-brand-colors-dynamic-theme-colors-
